.travis.yml

```diff
@@ -2,7 +2,10 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "iojs"
+  - "1"
+  - "2"
+  - "3"
+  - "4"
```

Comment:
Travis use nvm, so it should be supported.

See https://github.com/koajs/koa/blob/master/.travis.yml